src/index.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
+import './style.css';
+
+const input = document.getElementById('task');
+const submit = document.querySelector('.add-btn');
+const tasksDiv = document.querySelector('.tasks');
+
+let tasks = [];
+
+if (localStorage.getItem('tasks')) {
+  tasks = JSON.parse(localStorage.getItem('tasks'));
+}
+
+const addelemToPage = (tasks) => {
+  tasksDiv.innerHTML = '';
+  tasks.forEach((task, index) => {
+    const div = document.createElement('div');
+    div.className = 'task';
+    div.setAttribute('data-id', task.id);
+    div.innerHTML = `
+    <input id="${index}" class="checkbox" type="checkbox">
+    <p class='text' contenteditable="false">${task.descripton}</p>
```

Comment:
- [ ] When I edit a task the new description is not implemented in the local storage so when I reload the page the old description is what is still shown. Kindly fix that.